android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/tts/TTSServiceImpl.kt

```diff
@@ -2,14 +2,19 @@ package jp.hacks.smartbread.ui.main.tts
 
 import android.content.Context
 import android.speech.tts.TextToSpeech
+import java.util.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jphacks/FK_1908/blob/0d6e5b97259fa5aa77f3f0c6278426ea134696fd/android/Smartbread/app/src/main/java/jp/hacks/smartbread/ui/main/tts/TTSServiceImpl.kt#L5"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>